.github/workflows/aod-iam.yml

```diff
@@ -0,0 +1,51 @@
+# Workflow that validates and handles AOD IAM request.
+name: 'aod-iam'
+
+on:
+  pull_request:
+    types: ['opened']
+    branches:
+      - 'main'
```

Comment:
Do we need this?

Reply:
Why not?

Reply:
oh, you meant the pull_request event, I am thinking we should validate the request whenever a PR opened.

Reply:
I see. Then the validation should happen on every pull request push. Not just opened